aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/PersistentMutationOutbox.java

```diff
@@ -137,16 +132,7 @@ public boolean hasPendingMutation(@NonNull String modelId) {
     }
 
     private <T extends Model> void updateExistingQueueItemOrAppendNew(PendingMutation<T> pendingMutation) {
```

Comment:
After your refactor, you have several wrapper methods left inside of `PersistentMutationOutbox`, which are ~one line long. I would remove these, and just call your dat astructure directly, instead of through these wrapper methods:
- `updateExitingQueueItemOrAppendNew`
- `nextMutationForModelId`
- `removeFromQueue`
- `findPendingMutationById`

Reply:
@jamesonwilliams sure will revise